CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-## [1.3.0] - 2023-30-01
+## [1.3.x] - 2023-30-01
```

Comment:
@EmilianoMorghen it is not convenient to write this in the changelog, we have to indicate the changes of each version. in this case add the section for 1.3.2 indicating what changed